custom/icds_reports/tasks.py

```diff
@@ -641,6 +641,15 @@ def prepare_excel_reports(config, aggregation_level, include_test, beta, locatio
             loc_level=aggregation_level,
             show_test=include_test
         ).get_excel_data(location)
+        # as DatabaseColumn from corehq.apps.reports.sqlreport doesn't format None
```

Comment:
I think this should all be in `get_excel_data` (or one of the functions it calls but not in the download task like this)